Project-I-Social-Card/package.json

```diff
@@ -0,0 +1,5 @@
+{
+  "dependencies": {
+    "moment": "^2.23.0"
+  }
+}
```

Comment:
Try making sure your package.json looks like this and we will see if that fixes your moment dependency problem I encountered...

```
{
  "name": "social-card",
  "version": "0.1.0",
  "private": true,
  "dependencies": {
    "react": "^16.4.0",
    "react-dom": "^16.4.0",
    "react-scripts": "1.1.4",
    "moment": "^2.23.0"
  },
  "scripts": {
    "start": "react-scripts start",
    "build": "react-scripts build",
    "test": "react-scripts test --env=jsdom",
    "eject": "react-scripts eject"
  }
}
```